app/assets/javascripts/ui_models/app_state/files_state.ts

```diff
@@ -93,6 +93,10 @@ export class FilesState {
           ? await StreamingFileReader.getFilesFromHandles([fileOrHandle])
           : await picker.selectFiles();
 
+      if (!selectedFiles.length) {
```

Comment:
```suggestion
      if (selectedFiles.length === 0) {
```

Better to keep types consistent